be/src/storage/meta_reader.cpp

```diff
@@ -199,6 +231,10 @@ Status SegmentMetaCollecter::_collect_dict(ColumnId cid, Column* column, Logical
     auto dst = array_column->elements_column().get();
     CHECK(dst->append_strings(words));
 
+    if (column->is_nullable()) {
+        nullable_column->null_column_data().emplace_back(0);
```

Comment:
miss update_has_null ?

Reply:
> miss update_has_null ?

no need here, because 0 indicates not null, it won't change the result of has_null